90zfsbootmenu/lib/kmsg-log-lib.sh

```diff
@@ -3,77 +3,78 @@
 
 : "${loglevel:=4}"
 
-# arg1: log level
-# arg2: log line
+# arg1..argN: log line
```

Comment:
If we're embracing include guards, let's do so here too because we keep re-sourcing this file. However, we probably want this file to be reloadable (see comment on zfsbootmenu-parse-commandline.sh, so I recommend the following:
```sh
if [ -n "${_ZFSBOOTMENU_KMSG_LOG_LIB}" ]; then
  [ -z "${FORCE_RELOAD}" ] && return
else
  readonly _ZFSBOOTMENU_KMSG_LOG_LIB=1
fi
```

We could make the same change in `echo-log-lib.sh`, but we don't actually use that anywhere except in the `zbm-kcl` utility.